### DISCUSSION TITLE
Github counting more commits than there are

### DISCUSSION THREAD

**Discussion by claudi** [community/community#23158] [New to GitHub]
<p>Hello,</p>
<p>Today I made a lot of small contributions to a private repo, which totaled to around 80 contributions, while pushing them to github every so often. After this, I decided to squash some of them together and force pushed the result back to the github remote.</p>
<p>The problem is that now github thinks I made even more contributions, totaling to over 90. Going through the commits I have checked there are about 40 left after squashing, and I do not know what could have caused this, or how to fix it.</p>
<p>Any help would be appreciated.<br>
Thanks.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hey - without a viewable example this will be hard to troubleshoot.</p>
<p>If it’s a private repository, the I’d recommend you <a href="https://support.github.com/contact">open a ticket</a> so that it can be dealt with privately.</p>

**Comment by claudi** [comment]
<p>Thanks! I contacted them and they triggered a rebuild of the contribution for that repository, which fixed it.</p>
